pkgs/applications/video/vdr/default.nix

```diff
@@ -41,7 +41,11 @@
     mkdir -p $out/lib/vdr # only needed if vdr is started without any plugin
     mkdir -p $out/share/vdr/conf
     cp *.conf $out/share/vdr/conf
-    '';
+  '';
+
+  postFixup = ''
+    patchShebangs $out/bin/svdrpsend
```

Comment:
I believe the actual issue here is that `perl` is a `nativeBuildInputs`.
Can you see if moving `perl` to `buildInputs` fixes this for you? (removing this line also).

Reply:
You are right, adding `perl` to `buildInputs` fixed the issue. I kept `perl` also in `nativeBuildInputs` since it is used during building.